### DISCUSSION TITLE
Make 'o' continue the list in vim mode.

### DISCUSSION THREAD

**Discussion by platers** [vslinko/obsidian-outliner#194] [Ideas]
Currently `o` and `O` create a new indented blank line below or above the current list. It would be nice if it also inserted a bullet(`-`) to quickly add items to a list from vim mode.

**Comment by filyp** [comment]
Related: https://forum.obsidian.md/t/better-vim-support-in-lists/18826
I posted this to obsidian forum, because it seemed more like vim mode feature than obsidian-outliner feature, but no one responded :/
